packages/AutoTDD-Tests.package/ATDDNullDataSourceTests.class/instance/testSingleton.st

```diff
@@ -4,4 +4,7 @@ testSingleton
 	| dataSource |
 	
 	dataSource := ATDDNullDataSource instance.
-	self assert: dataSource equals: dataSource
\ No newline at end of file
+	self assert: dataSource equals: dataSource.
+	self assert: ATDDNullDataSource allInstances size <= 1
+	
+	"Todo in singleton tests: check if new/initialize throw errors"
```

Comment:
Remove comment (Unless I'm mistaken in that you haven't fixed all the singleton tests yet, but judging by your commit messages you seem to have done so)

Reply:
A thanks for pointing it out, was a note to future self to rewrite the tests :P